Dockerfile.server

```
@@ -7,21 +7,34 @@ WORKDIR /app
 
 FROM base as builder
 
-COPY package.json yarn.lock ./
+COPY package.json yarn.lock
 RUN yarn install --frozen-lockfile
 
 COPY . .
 
-RUN yarn lerna bootstrap
-RUN rm -rf /app/packages/server/dist/ && yarn build --scope=server
-
-
-FROM node:$NODE_VERSION-buster-slim as production
-WORKDIR /app
-
-COPY --from=builder /app/packages/server/dist/ /app/
-COPY --from=builder /app/packages/server/package.json /app/package.json
+RUN yarn bootstrap
+WORKDIR /app/packages/client
+RUN rm -rf /app/packages/client/dist && rm -rf /app/packages/client/ssr-dist
+RUN yarn build
+RUN yarn link
```

Можно сократить количество слоев образа, в теории написано, что это снижает вес образа. Но в данном случае размер не меняется.
```suggestion
RUN rm -rf /app/packages/client/dist && rm -rf /app/packages/client/ssr-dist  && \
yarn build  && \
yarn link
```